### ISSUE TITLE
[NPC] Unatackable Wastewalker Captive attacks players in Slave Pens

### ISSUE THREAD

**Issue by Fauchy** [dalaranwow/dalaran-wow#4067]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Several times in the last week I couldn't complete the dungeon "Slave Pens" on normal cause someone in our group did aggro up to three "Wastewalker Captives".
Since you can't damage those NPCs, the healer will get those mobs on himself cause of the threat generated by healing spells while keeping the attacked player alive. 
It's diffucult to handle this and will most times lead into a wipe.

You can damage those Captives if you have something like Retribution Aura (http://www.wowhead.com/spell=203797) or Thorns (http://www.wowhead.com/spell=33907) 
on yourself, while you get sucessful hit by a Captive. (No proof added)

**Expected behaviour**: Tell us what should happen instead

I am refering to another issue at this point:
#1819
"Also, on a more minor note, there's also an issue with Wastewalker Captive (http://www.wowhead.com/npc=18206). Those are 'doodad' mobs, their only purpose is to sit in cages there and say a following line: 'Help me please'. Player cannot interact with them in any way, though. There should be 2 or 3 of them in each cage."

**Steps to reproduce the problem**:
1. Enter Slave Pens with any character at lower leve(60).
2. Get close to the "Wastewalker Captive" NPC (May to the ones that are marked at the following sceenshot)

**Include proofs for this behaviour**
I will let this video talk for me:
https://vid.me/cTSC
When I turn the name templates on, I can't see the names of that Captive.
I am also using my Ghoul ability but it seems i can't damage that Captive. 

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Zone: Dungeon "Slave Pens" (http://www.wowhead.com/zone=3717)
NPC: Wastewalker Captive (http://www.wowhead.com/npc=18206)

**Include Screenshots from the issue if necessary**

http://i.imgur.com/5htr7VK.png
This is a map of the dungeon where I have marked the positions of the NPCs that got aggroed by players. May there are more.

**Comment by Rushor** [comment]
DB/Creature: Update faction and unitflags for 'Wastewalker Captive'

* closes https://github.com/dalaranwow/dalaran-wow/issues/4067
* commit: 4e68c906bc8dd2715ebe9ceafe5b5733d9a1656d

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by koubx** [comment]
Hi, 

Just to inform people, it's still an issue.

Following link is a folder with screenshots that show what could happen:
http://home.koub.org/files/slave-pens-bug/

Note that the one mob attacking is from the next big room (it came at us after we walked back after wipe). That is, from the link above (http://i.imgur.com/5htr7VK.png), the mob comes a bit bellow mark "5". 

Best regards.

**Comment by Piertotum** [comment]
Hi,

this is still an issue. 3 times we went in, 3 times this particular "Wastewalker Captive" second nearest to Quagmirran will start attacking the healers usually as she attracts aggro healing the party. Still unattackable.

The only work around so far is to let the healer die near the bugged NPC while the rest of the party stands far away. Any nearby character, even sitting down, will attract this NPC's aggro.

**Comment by Lothloryen** [comment]
Hello there,

It should be fine after the next realm maintenance, if anyone still notices any issue with these NPCs, feedback will be welcome.

Thank you.
